models/points.go

```diff
@@ -2462,9 +2475,10 @@ func appendField(b []byte, k string, v interface{}) []byte {
 	return b
 }
 
-// ValidKeyToken returns true if the token used for measurement, tag key, or tag
-// value is a valid unicode string and only contains printable, non-replacement characters.
-func ValidKeyToken(s string) bool {
+// ValidToken returns true if the provided token is a valid unicode string, and
+// only contains printable, non-replacement characters.
+func ValidToken(a []byte) bool {
+	s := string(a)
```

Comment:
There's a `uf8.Valid` that takes a `[]byte`. Given this is in a hot path, that's probably useful.

Reply:
dang, not sure how I missed that.